frontend/src/pages/associations/elections/Edit.tsx

```diff
@@ -0,0 +1,416 @@
+                    Il manque encore les registered voters pour l'instant 17bocquet et 17wan-fat par defaut
+                </Form.Row>
+                <Form.Row>
+                    <Form.Group>
+                        {(props.edit) ?
+                            <Button
+                                variant="danger"
+                                className={"mr-3"}
+                                type="button"
+                                onClick={deleteElection}
+                            >
+                                Supprimer
+                            </Button> : <></>
+                        }
+                        <Button
+                            variant="primary"
+                            type="button"
+                            onClick={handleSubmit}
+                        >
+                            {props.edit? "Supprimer et remplacer":"Créer l'élection"}
+                        </Button>
+                    </Form.Group>
+                </Form.Row>
+            </form>
+        </>
+    );
+};
+
+const ListOfChoices = props => {
+    const list = props.choices.map((choice, k) =>
+        <Col>
+            <ChoiceInput
+                id={k}
+                onChange={(e) => props.onChange(e, k)}
+                onDelete={() => props.onDelete(k)}
+                text={choice}
+            />
+        </Col>
+    );
+
+    const workOnList = (list: any[], n: number) => {
+        const lists : any[][]= [];
+        list.map((o, k) => {
+            if (k % n === 0) {
+                lists.push([o])
+            } else {
+                lists[lists.length -1].push(o)
+            }
+            return null;
+        });
+        return lists
+    };
+    const newList = workOnList(list, 3);
+
+    return (
+        <>
+            {newList.map(l =>
+                <Form.Row>
+                    {l}
+                </Form.Row>
+            )}
+        </>
+    )
+};
+
+const ChoiceInput = props => {
+    return (
+        <InputGroup className="my-2 form-control-g">
+            <Form.Control
+                id={"choice" + props.id}
+                name={"choice" + props.id}
+                type="text"
+                placeholder={'Entrez un choix'}
+                value={props.text}
+                onChange={(e) => props.onChange(e)}
+                required
+            />
+            <InputGroup.Append>
+                <Button
+                    onClick={() => props.onDelete()}
+                    variant={'secondary'}
+                >
+                    <i className="fe fe-trash-2"/>
+                </Button>
+            </InputGroup.Append>
+        </InputGroup>
+
+    )
+};
+
+const NewChoiceInputButton = (props) => {
+    return (
+        <Button
+            variant={'outline-dark'}
+            onClick={() => props.onClick()}
+        >
+            Nouvelle option
+        </Button>
+    )
+};
+
+const toGoodString = (date: Date) => {
```

Comment:
You should take a look at [String#padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) ;)

Reply:
It helped so much, thanks